src/utils.js

```diff
@@ -50,6 +50,7 @@ type CommentHeaders = 'Reviewers' | 'Required Reviewers' | 'Notified';
 export const makeCommentBody = (
     peopleToFiles: {[string]: Array<string>, ...},
     sectionHeader: CommentHeaders,
+    commentPerson: boolean = false,
```

Comment:
SUPER NIT: I've been preferring using an enum or string literal union for things like this as it's easy to see what it means at the callsite. Instead of having to work out what the `true` means, it can easily be read as something like `comment-person` versus `tag-person` or whatever.